ddio/lnxfile.cpp

```diff
@@ -493,6 +258,11 @@ void ddio_GetRootFromPath(const char *srcPath, char *root_path) {
   strcpy(root_path, "/");
 }
 
+std::vector<std::filesystem::path> ddio_GetSysRoots() {
+  // In Linux assume that all other mount points are accessible via root mount point
```

Comment:
this makes the extra column unnecessary on linux, should we hide it?

Reply:
I'd like keep unified UI layout in-code for a while, without of nasty `#ifdef`s. There a plenty of numbers should be recalculated otherwise.